spec/jobs/auto_enrol_job_spec.rb

```diff
@@ -7,17 +7,19 @@
 
       secondary_certificate = create(:secondary_certificate)
       primary_certificate = create(:primary_certificate)
+      subject_knowledge = create(:cs_accelerator)
```

Comment:
What is the reasoning for this variable being called `subject_knowledge` while the programme is also referred to as `cs_accelerator`? Are these names analogous? It seems inconsistent with the two lines above to use a different name but this isn't the only place that name is used like that so I don't have a strong feeling either way.

Reply:
cs_accelerator has become subject_knowledge, and its been changed in some places in but not others, I started out calling it subject knowledge, then realised afterwards it was still called cs_accelerator in the factory and just forgot to update the variable